prow/cmd/plank/main.go

```diff
@@ -69,27 +69,35 @@ func main() {
 		logrus.WithError(err).Fatal("Error starting config agent.")
 	}
 
-	oauthSecretRaw, err := ioutil.ReadFile(*githubTokenFile)
-	if err != nil {
-		logrus.WithError(err).Fatalf("Could not read oauth secret file.")
-	}
+	var oauthSecret string
+	var err error
+	if *githubTokenFile != "" {
```

Comment:
So you will pass `--github-token-file=''`?

Reply:
right.. yeah.. (given only the gerrit deployment is not using github right now)